process/block/baseProcess.go

```diff
@@ -1288,6 +1288,35 @@ func (bp *baseProcessor) DecodeBlockBody(dta []byte) data.BodyHandler {
 	return body
 }
 
+func (bp *baseProcessor) saveEpochStartInfoToStaticStorage(header data.HeaderHandler, marshalledHeader []byte, body *block.Body) {
+	epochStartBootstrapKey := append([]byte(common.EpochStartStaticBootstrapKeyPrefix), []byte(fmt.Sprint(header.GetEpoch()))...)
+	err := bp.store.Put(dataRetriever.EpochStartStaticUnit, epochStartBootstrapKey, marshalledHeader)
+	if err != nil {
+		log.Warn("saveEpochStartInfoToStaticStorage.Put header", "error", err.Error())
```

Comment:
is it intended to have the situation when we failed to save the header here, but we manage to save the miniblock in the following lines?

Reply:
added return here